pkg/api/config/config.go

```diff
@@ -68,6 +68,7 @@ type HTTPConfig struct {
 	Auth             *AuthConfig
 	RawAccessControl map[string]interface{} `mapstructure:"accessControl,omitempty"`
 	Realm            string
+	AllowOverwrites  bool             `mapstructure:",omitempty"`
```

Comment:
1. @ we though this new configuration should be under `HTTPConfig`. Is this ok, or do you think it should be under `GlobalStorageConfig`?
2. Please add an example of such a configuration under `zot/examples`